src/handlers/proposalVersionsHandlers.ts

```diff
@@ -45,7 +45,7 @@ const assertApplicationFormExistsForProposal = async (
 
 	let proposal: Proposal;
 	try {
-		proposal = await loadProposal(proposalId);
+		proposal = await loadProposal(undefined, proposalId);
```

Comment:
Passing `undefined` here and elsewhere feels wrong, somehow. Passing `null` would feel wonky too but at least we typically think of `null` as a potential value. Shouldn't `undefined` be reserved for cases when things are not defined?

When I tried to pass nothing by deleting `undefined` I of course got a typescript error because it expects an arg. When I tried to pass `null` it doesn't like that either because the generator does not generate `| null` as part of the signature. So I don't know if there's a quick fix here.

Is it possible to make the auth context an optional argument?

Reply:
Regarting optional: in this case I think auth context is so critical to think about that we should force devs to choose to explicitly not pass one.

That said, I'm completely fine with `null` instead of `undefined`; it sounds like that would address the root concern, right?

This would apply to the other generator too which is no problem at all.

Reply:
I think so. Is there any pitfall you thought of that is avoided by passing `undefined` instead of `null`? If it only feels better to me to pass `null` and there is some reason to pass `undefined` I can live with it.

Reply:
No I think you're right!  I think this is what `null` is for, and it doesn't seem like `undefined` gives any particular benefits (we can still do things like `??` with null just like with undefined)